### PR TITLE
[Ready] improvised glass hatchets

### PR DESCRIPTION
**Pull request by ghost**
[Changelogs]: 
:cl: zamolxius
add: Added improvised glass hatchets and it's recipe
imageadd: added it's sprites
/:cl:

[why]: just another improvised weapon

### CODE REVIEW

File: hippiestation/code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -137,4 +137,14 @@
...
 	force = 16
 	throwforce = 10
-	block_chance = 10
\ No newline at end of file
+	block_chance = 10
+
+/obj/item/hatchet/improvised
+	name = "glass hatchet"
+	desc = "A makeshift hand axe with a crude blade of broken glass."
+	flags_1 = null
```
Comment (ghost) [inline hippiestation/code/game/objects/items/weaponry.dm]: does not conduct electricity because it's glass

**Comment by JohnGinnane** [comment]
yessss

**Comment by praisenarsie** [comment]
tested and works?

**Comment by ghost** [comment]
yes

**Review by JohnGinnane — DISMISSED** [review]
Should this do less damage or leave it as is?

**Comment by ghost** [comment]
it does 12 brute, same as the botany hatchet of which there are 2 in arrivals' garden

**Comment by JohnGinnane** [comment]
yeah, I don't need any changes to damage are needed

**Comment by ghost** [comment]
technically its an objective upgrade though?
it doesn't conduct, is more easily replaceable and does the same damage

**Comment by JohnGinnane** [comment]
That's a very good point, this isn't a sidegrade or downgrade, it's a straight upgrade, so how would you balance it?

**Comment by ghost** [comment]
it's a really minor thing that i doubt is gonna be balance breaking but bringing it down from 12 force to 10 makes it take more hits to kill while still being a small tool that can cause bleed and be used as a hatchet

**Comment by JohnGinnane** [comment]
@zamolxius how does that sound?

**Comment by ghost** [comment]
how about making it conduct so it's still worthwhile as a weapon and not replaced by a fire extinguisher or something?

**Comment by JohnGinnane** [comment]
Could do that, would be a sidegrade then which is fine too, plus even though the blade is made of the glass the handle is still a metallic wrench

**Comment by GuyonBroadway** [comment]
Make it conduct and need duct tape. Duct tape is somewhat uncommon so it limits the amount that can be produced.

**Comment by ghost** [comment]
ad9fbb414fe9a935ebf6df654d54435b 1

**Comment by ghost** [comment]
tested

**Review by JohnGinnane — DISMISSED** [review]

**Comment by ghost** [comment]
ready 4 review

**Review by Carbonhell — APPROVED** [review]

**Comment by Carbonhell** [comment]
Sure +1

**Review by JohnGinnane — APPROVED** [review]

**Pull request merged**